playbooks/common/openshift-cluster/upgrades/upgrade_control_plane.yml

```diff
@@ -238,29 +238,22 @@
   any_errors_fatal: true
 
   pre_tasks:
+  - name: Load lib_openshift modules
+    include_role:
+      name: lib_openshift
```

Comment:
I agree that it is useful, especially when you consider the additional parameters that you can pass to override the task file, vars files, etc that are invoked from the role or if you want to have the role task execution happen at a specific time instead of relying on the role dependency order.

My concern is more around simply using it in place of normal role dependencies or the roles attribute of a play.

It may make complete sense to use include_role in pre_tasks here, but I don't think we should be arbitrarily using it everywhere.